IA/wiki_dump.py

```diff
@@ -0,0 +1,103 @@
+import os
+import math
+import asyncio
+import requests
+import argparse
+from ratelimit import sleep_and_retry
+from ratelimit.exception import RateLimitException
+from settings import SLEEP_PERIOD, OSF_API_URL
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+class WikiDumpError(Exception):
+    pass
+
+
+@sleep_and_retry
+def get_with_retry(url):
+    resp = requests.get(url)
+    if resp.status_code in (429, 500, 503):
```

Comment:
500 and 503 shouldn't be lumped in with 429; that's probably a legit error in those cases, though yes, one that might be solved with time. 

For the 429 case, can you pull the `retry-after` [response header](https://developer.osf.io/#tag/Errors-and-Error-Codes) value rather than using a configured value for sleep_period?